components/fs/nfs/op.c

```diff
@@ -25,13 +25,15 @@
 #include "util.h"
 #include "fd.h"
 
-#define MAX_CONCURRENT_OPS 100
+#define MAX_CONCURRENT_OPS FS_QUEUE_CAPACITY
 #define CLIENT_SHARE_SIZE 0x4000000
 
 struct fs_queue *command_queue;
 struct fs_queue *completion_queue;
 char *client_share;
 
+char path_buffer[4096][2];
```

Comment:
```suggestion
char path_buffer[FS_MAX_PATH_LENGTH][2];
```
?

Although that would be 4095 instead of 4096.

Reply:
It should be `FS_MAX_PATH_LENGTH + 1` now that I look at `copy_path`